src/toil/cwl/cwltoil.py

```diff
@@ -3111,7 +3141,11 @@ def hasChild(self, c: Job) -> Any:
 
 
 def remove_pickle_problems(obj: ProcessType) -> ProcessType:
-    """Doc_loader does not pickle correctly, causing Toil errors, remove from objects."""
+    """
```

Comment:
This function is only called in one place now, other than itself.  Is it safe to remove and was the issue resolved?

Reply:
I think it's still needed as I ran into this issue while creating the CWLImportJob. Something internal in the CWL tool object is unpickleable and must be removed. Before it was being called in every Job initialization, but now I moved it to be ran only once on the leader.